rfcs/test262_integration.md

```diff
@@ -0,0 +1,70 @@
+# RFC 229: Integrating Test262
+
+## Summary
+
+This RFC proposes integrating the Test262 (ECMAScript) test suite into WPT. Due to its large size (>50,000 files), tests will be fetched on-demand rather than vendored into the repository. Execution will be opt-in via a new `--test262` flag. This provides a unified way to run this conformance suite while protecting the WPT repository from bloat and performance degradation.
+
+## Motivation
+
+*   **Browser-based Testing:** Run Test262 in real browsers (stable and experimental), not just JS shells. This provides a more complete conformance signal for how ECMAScript behaves in the full web platform environment.
```

Comment:
I'd suggest "Allowing ECMAScript features to be proposed and selected in Interop" as a reason for wanting this.

Reply:
Done